### PR TITLE
Interactive Map - Issue#10

### PR DESCRIPTION
**Pull request by k-abirami**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

Added map functionality to index.html. Created a file called map.html and created a plugin location called locations.rb. Modified the all.scss file to add style to the map. We added markers to showcase the location of each fellow and added the profile picture for each fellow accordingly. We also added each fellow's name, location and description for each marker. The user would have to insert an API key from : https://developers.arcgis.com/sign-up/

There would be a potential merge conflict with issue#9, but their work on fellows.yml would override our fellows.yml file
image

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -37,7 +37,7 @@ GEM
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
-    liquid (4.0.3)
+    liquid (4.0.4)
```
Comment (akshgpt7) [inline Gemfile.lock]: You don't need to commit the Gemfile.lock. Can you please omit this file?

**Review by akshgpt7 — CHANGES_REQUESTED** [review]
Thank you for working on this!
However, I can't test it out and the map doesn't show the fellows in the preview. Can you find out the cause and fix it?

File: Gemfile.lock (hunk 1 above)
Comment (ThGreatExplorer) [reply Gemfile.lock]: Hi, did you generate an API key for Ersi Leaflet Api?

Btw, how do you want us to include API keys

**Comment by ttn-ttn** [comment]
Just an observation: you don't need the plugin. That plugin is reading data from site.data['fellows'] and writing it back completely unmodified to site.data['fellows_information']. You are already just using site.data.fellows in map.html, so you can just delete the plugin without any loss of functionality.

**Comment by akshgpt7** [comment]
Hey, is this ready to be reviewed? 
@k-abirami @ttn-ttn @k-abirami

**Comment by k-abirami** [comment]
> Hey, is this ready to be reviewed? @k-abirami @ttn-ttn @k-abirami

Yes, Will has also added the API key to the website

**Comment by akshgpt7** [comment]
@k-abirami I checked on Netlify, the API key has been added, it will probably work fine when it's merged, so can you just fix the conflicts and address the small change

**Comment by akshgpt7** [comment]
@k-abirami I checked on Netlify, the API key has been added, it will probably work fine when it's merged, so can you just fix the conflicts and address the small change

**Comment by akshgpt7** [comment]
@k-abirami I checked on Netlify, the API key has been added, it will probably work fine when it's merged, so can you just fix the conflicts and address the small change

File: index.html [hunk 2]
```diff
@@ -11,7 +11,10 @@
 
 {% include projects.html %}
 
+{% include map.html %}
+
 {% include footer.html %}
+
 </body>
 
 </html>
```
Comment (ttn-ttn) [inline index.html]: This two trailing `</body>` `</html>` tags aren't needed, as they are already in the layout used by index.html

**Comment by akshgpt7** [comment]
Looks great! Thanks for making this work!

**Comment by wrussell1999** [comment]
> @k-abirami I checked on Netlify, the API key has been added, it will probably work fine when it's merged, so can you just fix the conflicts and address the small change

API Key should be added now.

**Pull request merged**